catalog/ui/catalog-ui-search/src/main/webapp/component/dropdown/popout/dropdown.popout.view.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define*/
+var Marionette = require('marionette');
+var _ = require('underscore');
```

Comment:
Nope, I must have forgotten to drop this.  It won't really bloat the UI at all in this case since underscore is required very early on in the UI tree.  But if it was a library that no other component used, it definitely would cause it to be packaged and delivered too early in webpack.